ext/date/config0.m4

```diff
@@ -10,7 +10,7 @@ timelib_sources="lib/astro.c lib/dow.c lib/parse_date.c lib/parse_tz.c lib/parse
 
 PHP_NEW_EXTENSION(date, php_date.c $timelib_sources, no,, $PHP_DATE_CFLAGS)
 
-PHP_ADD_BUILD_DIR([$ext_builddir/lib], 1)
+PHP_ADD_BUILD_DIR([$ext_builddir/lib], [1])
```

Comment:
What is the "create" argument required here actually?

Reply:
Because it creates ext/date/lib/timelib_config.h few calls later using cat command so when building in out-of-source directory (directory different than the php-src) the the lib directory must already exist in the ext/date.

Reply:
ACK